App/Selectors/UserAuthSelector.js

```diff
@@ -0,0 +1,11 @@
+import { createSelector } from 'reselect';
+
+const selectUserAuthDomain = state => state.userAuth;
+
+export const selectSavedUserData = createSelector(
+  selectUserAuthDomain, state => state.data
+);
+
+export const selectIsUserAuth = createSelector(
```

Comment:
`isUserAuth` doesn't say much. Does it mean that user is authenticated? There's no need to shorten this name